lib/pure/nativesockets.nim

```diff
@@ -482,8 +483,18 @@ proc getAddrString*(sockAddr: ptr SockAddr): string =
         raiseOSError(osLastError())
     setLen(result, len(cstring(result)))
   else:
+    when defined(posix) and not defined(nimdoc):
+      if sockAddr.sa_family.cint == nativeAfUnix:
+        return "unix"
```

Comment:
I still don't think this is correct. What is using this proc that you need to add this in here?

Reply:
I agree, I also still don't think this is correct. But I do not know what the proper alternative would be.

This function is only used by accept(), but it seems that the semantics for sockaddr_un are not well defined on accept(). In my tests (linux 4.18+glibc2.27) the sun_path is simply not filled in after accept.

Reply:
hrm, alright. That makes sense, this will do for now.

Thank you for working on this!